app/assets/stylesheets/hera/variables.scss

```diff
@@ -26,6 +26,7 @@ $text-error: $red;
 $text-link: $brand-700;
 $text-link-hover: darken($text-link, 15%);
 $text-muted: $grey-700;
+$text-small: 0.9rem;
```

Comment:
can't we just use bootstrap's `small` class as needed?

Reply:
We definitely could. Sometimes I find it easier to bundle it together instead of going through each item to add the class. I'm happy to remove it if you find it unnecessary